homeassistant/components/litejet/switch.py

```diff
@@ -50,6 +61,11 @@ def name(self):
         """Return the name of the switch."""
         return self._name
 
+    @property
+    def unique_id(self):
+        """Return a unique identifier for this switch."""
+        return f"{self._entry_id}_{self._index}"
```

Comment:
There is a bit of code duplication. It might make sense in a followup PR to create a base class for litejet entities.